Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml

```
@@ -571,6 +581,17 @@
     sprite: Structures/Storage/Crates/trashcart.rsi
   - type: Sprite
     sprite: Structures/Storage/Crates/trashcart.rsi
+    layers:
+    - state: base
+    - state: closed
+      map: ["enum.StorageVisualLayers.Door"]
+    - state: welded
+      visible: false
+      map: ["enum.WeldableLayers.BaseWelded"]
+    - state: paper
+      sprite: Structures/Storage/Crates/labels.rsi
+      offset: "0.0,0.03"
```

```suggestion
      offset: "0.0,0.03125"
```

needs to be multiples of the pixel size (1 / 32)